content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -441,7 +437,7 @@ can handle to serving the `cluster-info` ConfigMap.
 Kubeadm installs the internal DNS server and the kube-proxy addon components via the API server.
 Please note that:
 
-1. This phase can be invoked individually with the [`kubeadm alpha phase addon all`](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-phase-addon) command.
+1. This phase can be invoked individually with the [`kubeadm init phase addon`](/docs/reference/setup-tools/kubeadm/kubeadm-alpha/#cmd-phase-addon) command.
```

Comment:
this probably still needs `all`

Reply:
+1

`kubeadm-alpha/#cmd-phase-addon` seems to be the wrong link, too. will fix.